package.json

```diff
@@ -2,12 +2,12 @@
   "name": "protosaurus",
   "version": "1.0.0",
   "private": true,
-  "workspaces":[
-    "website/*",
+  "workspaces": [
+    "website",
```

Comment:
Changing this to just `website` because `website` is no longer a workspace.

Reply:
Yes, and put `private: true` in website's package.json?